grid-proxy/internal/explorer/converters.go

```diff
@@ -11,16 +11,31 @@ import (
 )
 
 const (
-	nodeUpInterval = -80 * time.Minute
+	// node report its state every 40 mins, if didn't report for 2 cycles, it's marked down
+	// nodes powered by farmerbot report every 24 hours, if didn't report for 1 cycle, it's marked down
+	nodeUpStateFactor         = 2
```

Comment:
These variables are repeated throughout the code, perhaps they should be moved to their own module, and all shared logic can be moved there as well, to make sure that it is consistent across the code base